src/routes/(authenticated)/dashboard/[conferenceId]/DelegationRegistrationStage.svelte

```diff
@@ -122,7 +123,7 @@
 				newHeadDelegateUserId: userId
 			})
 		);
-		invalidateAll();
+		await invalidateAll();
```

Comment:
We should consider using fine grained invalidate calls for faster updates: https://kit.svelte.dev/docs/modules#$app-navigation-invalidate but this is fine for now

Reply:
Yes, did stumble across that as well. Goes into the chapter performance optimization and telemetry